echoshader/dev/echo_curtain.py

```diff
@@ -0,0 +1,50 @@
+import numpy
+import panel
+import pyvista
+
+panel.extension("vtk")
+"""
+MVBS_ds: with a specific freq
+"""
```

Comment:
```suggestion
```
This docstring should be placed within the function not at the top of the module